app/build.gradle

```diff
@@ -121,6 +121,7 @@ android {
 aboutLibraries {
     prettyPrint = true
     configPath = 'config/aboutLibraries'
+    excludeFields = arrayOf("generated")
```

Comment:
@licaon-kter: This is the wrong syntax because our `build.gradle` file uses Groovy and not Kotlin. The correct syntax should be `excludeFields = ['generated']`.